common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -599,6 +599,7 @@ class @Problem
         $p = $(element).find('span.status')
         `// Translators: the word unanswered here is about answering a problem the student must solve.`
         $p.parent().removeClass().addClass "unsubmitted"
+        $(element).find('.status .status-icon').remove()
```

Comment:
Have you been able to test these removes? Are they necessary (aren't we already re-rendering the whole thing on reset)?